### DISCUSSION TITLE
Differences between structmap and serde?

### DISCUSSION THREAD

**Discussion by after-ephemera** [ex0dus-0x/structmap#12] [Q&A] [answered]
According to [this issue comment](https://github.com/serde-rs/serde/issues/1739#issuecomment-585442986), serde's `MapDeserializer` can deserialize arbitrary maps into structs. Is there an advantage to using structmap besides the more limited scope and smaller crate size?

**Comment by ex0dus-0x** [comment, marked as answer]
Thanks for pointing this out!

I would say you are also right with the reasons why this crate exists: I needed something that had a smaller footprint, and the also the ability to rename field names during serialization. 

I'm also certainly not always on top of pushing out more features, so apologies on that front!
